### ISSUE TITLE
API error

### ISSUE THREAD

**Issue by ivanfig** [boyan-soubachov/tastyworks_api#23]
on the line:

```
orders = await Order.get_remote_orders(session, acct)
    LOGGER.info('Number of active orders: %s', len(orders))
```

I get error:

2018-09-09 22:24:59,161 Exception in main loop
Traceback (most recent call last):
  File "tasty_example.py", line 85, in main
    loop.run_until_complete(main_loop(tasty_client, streamer))
  File "C:\Users\idf\Anaconda\lib\asyncio\base_events.py", line 468, in run_until_complete
    return future.result()
  File "tasty_example.py", line 34, in main_loop
    orders = await Order.get_remote_orders(session, acct)
  File "C:\Users\idf\Anaconda\lib\site-packages\tastyworks\models\order.py", line 126, in get_remote_orders
    order = cls.from_dict(order_data)
  File "C:\Users\idf\Anaconda\lib\site-packages\tastyworks\models\order.py", line 84, in from_dict
    details.price = float(input_dict['price'])
KeyError: 'price'

**Comment by boyan-soubachov** [comment]
Thanks for the heads-up. Will have a look.

**Comment by boyan-soubachov** [comment]
I'm having a hard time reproducing the error. It seems to be some edge-case with tastyworks that I have not encountered previously.

Do you mind providing me with some more information, i.e. what the order details it's trying retrieve are? This error would happen if tastyworks doesn't report a price with the order which to me seems like a requirement.

**Comment by bonkeydcow** [comment]
I just installed and ran the example code. I get this exact error. I changed nothing except the login credentials.

**Comment by ivanfig** [comment]
It happens intermittently to me without changing anything. But it works for
me for now.

**Comment by boyan-soubachov** [comment]
Thank you, I'll have a look as soon as I'm able to.

**Comment by boyan-soubachov** [comment]
This issue should be resolved in the latest release (3.1.1). Please let me know if you get it again.
